### DISCUSSION TITLE
AMD 7 Ryzen 5800U (Radeon Vega 8 iGPU)

### DISCUSSION THREAD

**Discussion by keysteal72** [isc30/ryzen-gpu-passthrough-proxmox#60] [General]
I've been working on how use the GPU of my MiniPC (AM06PRO) for days and days, reading many articles and posts. The only way to passthrough this GPU (Amd Ryzen 5800U) was thanks of your article. I exported the vbios using your program, and I also use the rom for Audio, and voilà. The only thing that I can not understand, the passthrough only work if I use Seabios instead UEFI (Proxmox 8.2.2 boot with Uefi).

**Comment by keysteal72** [comment]
Anyway too many issues and bug, I will return this Computer (i've time to do). I raise a white flag :(

**Comment by Lky** [comment]
Hello could you provide the files you extracted for you CPU/GPU system, please? (vbios)

Maybe others wanna give it a try, thank you.

**Comment by keysteal72** [comment]
Attached here the Vbios for AMD 7 Ryzen 5800U (iGpu Radeon Vega 8)
[vbios_5800U.bin.zip](https://github.com/user-attachments/files/16652410/vbios_5800U.bin.zip)
